homeassistant/components/mqtt/const.py

```diff
@@ -36,6 +36,7 @@
 CONF_TLS_VERSION = "tls_version"
 
 DATA_MQTT = "mqtt"
+MQTT_DATA_DEVICE_TRACKER_LEGACY = "mqtt_device_tracker_legacy"
```

Comment:
I don't see what adding this const should solve. It does not seem to me it is referenced by other integrations.

Reply:
May be a custom integration references `MQTT_DATA_DEVICE_TRACKER_LEGACY` and fails to load. It was used to store internal MQTT legacy device tracker info. The legacy device tracker was removed with HA Core 2022.12 and the constant was removed. Any references to this old code should be cleaned up.

Reply:
See linked issue. The cause that this constant is imported is due to incidental installation errors that might occur where a code file that was removed from the source was not removed on the installation. We cannot support supplying a work-a-round for this. I tried a pip upgraded and concluded when pip is used correctly this will remove the old files (see linked issue). This source installation files (pip packages, docker images etc) are not having issues.
Closing this PR, because no change is needed.